test_docs.py

```diff
@@ -75,6 +75,12 @@
     inside_bash_code_block = False
     contents = ""
     for line in f.readlines():
+        if line.startswith("git clone"):
+            int_branch_env_var_name = "INTEGRATION_BRANCH"
+            integration_branch = os.environ.get(int_branch_env_var_name)
```

Comment:
If it is only used here, there is much value on having it as a variable.
```suggestion
            integration_branch = os.environ.get("INTEGRATION_BRANCH")
```

Reply:
That is how it was originally done, but it doesn't pass python formatting check.